<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductionWindow</class>
 <widget class="QMainWindow" name="ProductionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 44, 50), stop:1 rgb(49, 97, 117));
color:white;
font-family:Microsoft YaHei;
text-align: center;
}

QFrame{
border:sold 10px rgba(0 ,0, 0)
}


QLineEdit{
background-color:rgb(58. 104. 148);
}

QPushButton{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-color:white;
border-radius:3px;
border:0px solid ;
}

QPushButton:pressed{
background-color:QLinearGradient(x1:0, y1: 0, x2: 1, y2: 0, stop: 0 rgb(71,191,255) , stop: 1 rgb(23, 78, 115));
border-style:solid;
border-radius:3px;
border:1px solid;
}



QLineEdit{
border: 1px solid #A0A0A0;
background-color:rgb(65, 65, 65);
color:white;
}


QLineEdit:hover { 
    border: 1px solid #298DFF;
    border-radius: 3px;
    selection-color: #F2F2F2;
}

QLineEdit[EchoMode=&quot;2&quot;] {lineedit-password-character:42;}

QMenu::item 
{
     background-color:rgb(42, 42, 42);
}
 
/*选择项设置*/
QMenu::item:selected 
{
     background-color: rgb(36, 58, 99);
}
QMenuBar::item:selected{
background-color:rgb(35, 60, 91);
color:white;
}
QMenuBar::item{
color:rgb(134, 134, 134);
}
QTabWidget::pane{
border:none;
background: transparent;

}
QTabWidget::tab-bar {
border: none;   
}
QTabBar::tab {
border: none;
color: rgb(158, 158, 158);
font:22px;
background:transparent;
height: 28px;
min-width: 85px;
margin-right: 5px;
padding-left: 5px;
padding-right: 5px;

}
QTabBar::tab:hover {
background: transparent;
color:rgb(225, 225, 225);

}
QTabBar::tab:selected {
color: rgb(255, 255, 255);
background: transparent;
}

QLabel{
background-color:rgb(58, 82, 127);
}
#profileTabWidget::pane{
border:2px solid rgb(59, 98, 134);
border-radius:10px;
}

QStatusBar QLabel{
border:none;
background:transparent;
}

QCheckBox::indicator, QRadioButton::indicator, QTreeView::indicator {
width: 16px;
height: 16px;
background-color: #111111;
border: 1px solid #333333;
}

QRadioButton::indicator {
border-radius: 8px;
}

QCheckBox::indicator::checked, QRadioButton::indicator::checked, QTreeView::indicator::checked {
background-color: qradialgradient(cx:0.5, cy:0.5, fx:0.25, fy:0.15, radius:0.3, stop:0 #BBBBBB, stop:1 #111111);
}

QCheckBox::indicator:disabled, QRadioButton::indicator:disabled, QTreeView::indicator:disabled {
background-color: #444444;
}

QCheckBox::indicator::checked:disabled, QRadioButton::indicator::checked:disabled, QTreeView::indicator::checked:disabled {
background-color: rgb(0, 99, 148)
}

QLabel{
background:transparent;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="7,2,4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1">
        <item>
         <widget class="QLabel" name="bigPicLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="priceLabel">
          <property name="font">
           <font>
            <family>Microsoft YaHei</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
color:rgb(231, 21, 52)
}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="storageLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="freightLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>99999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="unitLabel">
            <property name="text">
             <string>Unit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="limitationLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,1,2,2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cartPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>加入购物车</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="orderPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>立即购买</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>商品介绍：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="introductionBrowser"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="commentWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
